neuro_data/static_images/data_schemas.py

```diff
@@ -251,7 +260,7 @@ def fill(self, scan_key):
         n = int(np.median(unique_frames.fetch('repeats')))  # HACK
         num_oracles = len(unique_frames & 'repeats > {}'.format(n))  # repeats
         if num_oracles == 0:
-            self.msg('Could not find repeated frames. Using 20% of the original set')
+            print('Could not find repeated frames. Using 20% of the original set')
```

Comment:
Shall we add a choice here for the user? Like 'Could not find repeated frames. Do you want to use 20% of the original set as oracle images? y/n'.

Reply:
i don't think so, the problemmwith asking users stuff is that it cannot be run as a a script anymore

Reply:
changed it to raise an Error instead